Cakefile

```diff
@@ -166,6 +169,30 @@ task 'build:browser', 'rebuild the merged script for inclusion in the browser',
   console.log "built ... running browser tests:"
   invoke 'test:browser'
 
+task 'build:inline_runtime', 'build the inline iced3 runtime', ->
+  runtime_dir = path.dirname require.resolve 'iced-runtime-3'
+  code = ''
+  for name in ['const', 'runtime', 'library', 'main']
+    code += """
+      require['./#{name}'] = (function() {
+        var exports = {}, module = {exports: exports};
+        #{fs.readFileSync "#{runtime_dir}/#{name}.js"}
+        return module.exports;
+      })();
+    """
+  code = """
+    (function() {
+      function require(path){ return require[path]; }
+      #{code}
+      return require['./main'];
+    }());
+  """
+
+  unless process.env.MINIFY is 'false'
+    {code} = require('uglify-js').minify code, fromString: true
+
```

Comment:
But this is the point of this branch, no? To emit ES6 for future browsers or as input for babel translation.

We can optionally support browserify in compiler - but do not actually specify it as a dependency - so it will simplify the workflow for people who have it installed anyways.